### ISSUE TITLE
[bug] webview.reload() creates history items

### ISSUE THREAD

**Issue by felicienfrancois** [atom/electron#1422]
The expected behavior (like other browser behave) is
- Reloading does not creates history items
- redirects does not creates history items

In atom shell:

```html
<webview id="foo" src="http://www.github.com/" style="width:640px; height:480px"></webview>
<script>
    var webview = document.getElementById("foo");
    // wait for page being loaded
    webview.canGoBack(); // returns false  
    webview.reload();
    // wait for page being loaded
    webview.canGoBack(); // returns true but should returns false
</script>
```

**Comment by zcbenz** [comment]
This is a duplicate of #1249.

**Comment by suchafreak** [comment]
Why was this bug closed and marked as a duplicate of  #1249?

The problem still exists in version 1.7.9: calling reload() on webview adds an item to the history as described by @felicienfrancois

**Comment by TheHtmler** [comment]
Hi, is this issue be resolved?I got the same situation in Electron 5.0.13. It always be loaded in each action when use location.forward or location.back if ever used location.reload before.
